README.md

```diff
@@ -1,160 +1,81 @@
 
-|MIP № |Date Ratified | Proposal Link                     |Description/Title                                  
-|------|--------------|-----------------------------------|---------------------------------------------------|
-|   1  |   2020-05-02 | [MIP1](https://github.com/makerdao/mips/tree/master/MIP1)                          |Maker Governance Paradigms  |
-|   2  |   2020-05-02 | [MIP2](https://github.com/makerdao/mips/tree/master/MIP2)                          |Launch Period                |
-|   3  |   2020-05-02 | [MIP3](https://github.com/makerdao/mips/tree/master/MIP3)                          |Governance Cycle                 |
-|   4  |   2020-05-02 | [MIP4](https://github.com/makerdao/mips/tree/master/MIP4)                          |MIP Amendment and Removal Process |
-|   5  |   2020-05-02 | [MIP5](https://github.com/makerdao/mips/tree/master/MIP5)                          |Emergency Voting System         |
+# The MIP Framework Overview
 
+## Definitions
 
-### Collateral Onboarding Framework
+### MIPs
 
+MIPs are standardized documents subject to community feedback and voting that, once enacted, define the behavior of the Maker Governance and the Maker Protocol. Collectively, MIPs are the dynamic body of law that regulates MakerDAO at any given moment.
 
-|MIP № |Date Ratified | Proposal Link                     |Description/Title                                  
-|------|--------------|-----------------------------------|---------------------------------------------------|
-|   6  |   2020-05-02 | [MIP6](https://github.com/makerdao/mips/tree/master/MIP6)                          |Collateral Onboarding Form/Forum Template |
-|   7  |   2020-05-02 | [MIP7](https://github.com/makerdao/mips/tree/master/MIP7)                          |Domain Teams (Collateral Onboarding)  |
-|   8  |   2020-05-02 | [MIP8](https://github.com/makerdao/mips/tree/master/MIP8)                          |Domain Greenlight (Collateral Onboarding)|
-|   9  |   2020-05-02 | [MIP9](https://github.com/makerdao/mips/tree/master/MIP9)                          |Community Greenlight (Collateral Onboarding) |
-|   10 |   2020-05-02 | [MIP10](https://github.com/makerdao/mips/tree/master/MIP10)                         |Oracle Management |
-|   11 |   2020-05-02 | [MIP11](https://github.com/makerdao/mips/tree/master/MIP11)                         |General Risk Model Management (Collateral Onboarding) |
-|   12 |   2020-05-02 | [MIP12](https://github.com/makerdao/mips/tree/master/MIP12)                         |Collateral and Risk Parameter Management |
+MIPs have two types: **technical** if they propose changes or additions to smart contracts code related to the Maker Protocol; or **general** otherwise.
 
+### Subproposals
 
-## Join the conversation 
+Subproposals are meta-processes of general, recurring processes defined in MIPs. Subproposals are created and used as a means of creating a process to build a process.
```

Comment:
Most people will not understand what you mean when using the phrase "meta-processes" & may also be confused by the "used as a means of creating a process to build a process" bit.

Here's my proposal to make it more simplified, let me know if you feel like this is not technically accurate in some way;

"Subroposals are a common recurring process found in numerous MIPs. They are created and used as a means of proposing and enacting decisions nested within specific MIPs."

What do you think @CPSTL @blimpa

Reply:
First two words that come to my mind are "instance [of a process]" and "actualization", both of which are probably not much of a improvement over "meta-processes"? Maybe something like "documents upon whose approval processes are executed" or some other paraphrase?

Reply:
"Subroposals are a common recurring process. They are created and used as a means of proposing and enacting decisions nested within specific MIPs."